README.md

```diff
@@ -117,3 +117,15 @@ Refer to the [Dev Guide][devguide] for development and debugging instructions.
 [knownissues]:          https://azure.github.io/iotedge-lorawan-starterkit/dev/issues/
 [partnerinstructions]:  https://azure.github.io/iotedge-lorawan-starterkit/dev/user-guide/partner/
 [specificationsupport]: https://azure.github.io/iotedge-lorawan-starterkit/dev/#lorawan-specification-support
+
+## Create a release
+
+You can create a prerelease with the following steps:
+
+- Run the Prerelease workflow
+  Go to `Actions`, select `Prerelease` action, then select `Run workflow` and specify the release version before running the workflow.
+- Merge 2 PRs
+  The Prerelease workflow will create 2 PRs, one PR to update the Starter Kit version in Bicep and one PR to update the Button URL. Merge these 2 PRs.
+- Add a release description and set as the latest
```

Comment:
I am not sure I understand here, Do you want to say "create a git tag pointing to the latest commit on the master branch"?

Reply:
In this case there is a step earlier, which is pushing dev to master